README.md

```diff
@@ -1,2 +1,21 @@
+![Logo](Resources/logo-black-transparent.png)
 # Aquarius
-Contains the source code for Aquarius, a 2D game engine for the ECE 5010 junior software design project
+A 2D game engine developed by Colton Smith, Daniel Burke, & Liam Warren for the Junior Software Design Project (ECE 5010).
+
+## Additional Required Software
+- CMake Version 3.10 ([Download](https://cmake.org/download/))
+
+## Building Aquarius
```

Comment:
Mention that these commands and the application work for Windows, MAC, and Linux, but not WSL (windows subsystem for linux)